src/index.html

```diff
@@ -1,17 +1,156 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="style.css"
+      rel="stylesheet"
+      type="text/css"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="#" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input type="text"
+            name="surname"
+            id="surname"
+            required
+            minlength="100"
+            maxlength="1000"
+          >
+        </div>
+        <div class="form-field">
+          <label for="name">
+            Name:
+          </label>
+          <input type="text"
+            name="name"
+            id="name"
+            required
+          >
+        </div>
+        <div class="form-field">
+          <label for="howOldAreYou">
+            How old are You?
+          </label>
+          <input type="number"
+            name="howOldAreYou"
+            id="howOldAreYou"
+            value="12"
+            min="1"
+            max="100"
+          >
```

Comment:
Please, fix the code style according to the checklist, everywhere 
```suggestion
          </label>
          
          <input 
            type="number"
            name="howOldAreYou"
            id="howOldAreYou"
            value="12"
            min="1"
            max="100"
          >
```